specs/latest/1.0/index.html

```diff
@@ -4455,6 +4459,17 @@ <h3>Wide point primitive clipping</h3>
     </p>
 </div>
 
+<h3>Undefined Point Size</h3>
+<p>
+    Draw calls with <code>POINTS</code> primitives are rejected when the bound program does not set
+    <code>gl_PointSize</code>.
+</p>
+<div class="note rationale">
+    <p>
+        Point rendering with undefined point size is undefined in GLES and may be unsupported in other APIs.
```

Comment:
From concall: please mention something more precise about why this doesn't work.

Reply:
After looking into this again, it seems there are two underlying issues there:
- D3D11 does not support point rendering with point size greater than `1.0`, so ANGLE generates a geometry shader to handle that.
- The presence of `gl_PointSize` in the original ESSL source non-trivially affects ANGLE's shader translator (e.g., how varyings are packed), so simply prepending all shaders with `gl_PointSize = 1.0;` may cause unexpected issues. Those may be caused by unknown bugs in the translator.

So this issue looks like an ANGLE limitation, which may remain indefinitely.

What would be the appropriate language for the spec here?